### ISSUE TITLE
setupWindowMock is not mocking window.location.href

### ISSUE THREAD

**Issue by elwayman02** [simonihmig/ember-window-mock#586]
When upgrading from v0.8.1 to v1.0.2, we're experiencing a regression in mocking window.location.href.

```
import window from 'ember-window-mock';
import { setupWindowMock } from 'ember-window-mock/test-support';

// ...

window.location.href = 'https://www.foobar.com'
```

When href gets set, the test window in the browser is transitioning to foobar.com, even though the README states that it's expected to do nothing when setting the mock (other than update the value).

**Comment by simonihmig** [comment]
I assume this is in a test, that has `setupWindowMock(hooks)` called in its parent `module`? 

Again here, this has [test coverage](https://github.com/simonihmig/ember-window-mock/blob/master/test-app/tests/unit/run-window-tests.ts#L72), so I tend to assume something specific to your usage is in play here?

One thing you can look into is to `console.log(window.location)`. If it's the mocked one, it just show up as a `Proxy` rather than `Location`.

**Comment by elwayman02** [comment]
Yes, `setupWindowMock(hooks);` is called in the parent module, before `hooks.beforeEach()`. Interestingly, window.location is NOT showing up as a Proxy, and even though I'm calling `setupWindowMock`, I can't hit a breakpoint inside the method itself.

I inspected setupWindowMock itself and got this:

```
> _testSupport.setupWindowMock
< ƒ setupWindowMock(hooks=self){hooks.beforeEach(()=>(0,_index_js__WEBPACK_IMPORTED_MODULE_1__._setCurrentHandler)((0,_window_js__WEBPACK_IMPORTED_MODULE_0__.createWindowProxyHandler)()));hooks.afterEach…
```

**Comment by elwayman02** [comment]
Here's what I have:

```
import { module, test } from 'qunit';
import window from 'ember-window-mock';
import { setupWindowMock } from 'ember-window-mock/test-support';

module('Unit | Util | some-util', function (hooks) {
  setupWindowMock(hooks);

  test('injectWalkmeBundle works for other countries', function (assert) {
    window.location.href = 'https://www.foobar.com';
    // At this point, `window.location` is not a proxy
  });
```

Package verisons:
ember-window-mock v1.0.2
ember-auto-import v2.6.3
ember-source v4.12.3
ember-cli v4.12.2
ember-qunit v6.2.0
@embroider/macros v1.16.6
